jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/request_handlers.py

```diff
@@ -0,0 +1,23 @@
+from notebook.base.handlers import IPythonHandler
```

Comment:
These handlers go along with the new mechanism for retrieving hither results. When a job finished, a message is sent from server to client, but that message no longer contains the actual result data (which may be large). Rather it contains the sha1 hash of the result object. The client then can make an async http request to retrieve the data through the /sha1/<sha1> url... this serves the purpose of not blocking the message-passing socket while large files are retrieved, and allows multiple result files to be transferred simultaneously (and potentially cached by the browser).